test/framework/src/main/java/org/elasticsearch/index/engine/DocIdSeqNoAndTerm.java

```diff
@@ -20,24 +20,34 @@
 package org.elasticsearch.index.engine;
 
 
+import org.apache.lucene.util.BytesRef;
+
 import java.util.Objects;
 
 /** A tuple of document id, sequence number and primary term of a document */
 public final class DocIdSeqNoAndTerm {
```

Comment:
we'll have to rename this class now. Perhaps `DocIdSeqNoAndSource`. It's not fully precise either, but gives a better overview of what's contained